### ISSUE TITLE
Main tag missing in html body & accessibility issue with labels

### ISSUE THREAD

**Issue by kalbek** [ehmaddd/mobile-first-portfolio#11]
## Organize HTML inside main element
In order to organize HTML contents in a better way and in order to be more semantically correct I recommend encapsulating the part of the code under the header inside the main element, and please remove the div on line 14
https://github.com/ehmaddd/mobile-first-portfolio/blob/a3cca31f4d804baaf7c00f9f9a4b7fccfaeba97a/index.html#L14

## Use clip property to hide HTML elements
also please consider hiding the labels on line 167 using clip property rather than display: none in your CSS
https://github.com/ehmaddd/mobile-first-portfolio/blob/a3cca31f4d804baaf7c00f9f9a4b7fccfaeba97a/index.html#L167
https://github.com/ehmaddd/mobile-first-portfolio/blob/a3cca31f4d804baaf7c00f9f9a4b7fccfaeba97a/styles.css#L691-L694
you can replace this CSS line with:
```
 .labels {
    border: 0;
    clip: rect(1px 1px 1px 1px);
    height: 1px;
    margin: -1px;
    overflow: hidden;
    padding: 0;
    position: absolute;
    width: 1px;
  }
```

**Comment by ehmaddd** [comment]
Yes accepted. It will be surely settled in the next update.

**Comment by touredri** [comment]
Good job

**Comment by Kwaku28** [comment]
Great job so far 👍
